test/e2e/job.go

```diff
@@ -96,120 +92,6 @@ var _ = framework.KubeDescribe("Job", func() {
 		err = waitForJobFinish(f.ClientSet, f.Namespace.Name, job.Name, completions)
 		Expect(err).NotTo(HaveOccurred())
 	})
-
-	It("should keep restarting failed pods", func() {
-		By("Creating a job")
-		job := newTestJob("fail", "all-fail", v1.RestartPolicyNever, parallelism, completions)
-		job, err := createJob(f.ClientSet, f.Namespace.Name, job)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Ensuring job shows many failures")
-		err = wait.Poll(framework.Poll, jobTimeout, func() (bool, error) {
-			curr, err := getJob(f.ClientSet, f.Namespace.Name, job.Name)
-			if err != nil {
-				return false, err
-			}
-			return curr.Status.Failed > lotsOfFailures, nil
-		})
-	})
-
-	It("should scale a job up", func() {
-		startParallelism := int32(1)
-		endParallelism := int32(2)
-		By("Creating a job")
-		job := newTestJob("notTerminate", "scale-up", v1.RestartPolicyNever, startParallelism, completions)
-		job, err := createJob(f.ClientSet, f.Namespace.Name, job)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Ensuring active pods == startParallelism")
-		err = waitForAllPodsRunning(f.ClientSet, f.Namespace.Name, job.Name, startParallelism)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("scale job up")
-		scaler, err := kubectl.ScalerFor(batchinternal.Kind("Job"), f.InternalClientset)
-		Expect(err).NotTo(HaveOccurred())
-		waitForScale := kubectl.NewRetryParams(5*time.Second, 1*time.Minute)
-		waitForReplicas := kubectl.NewRetryParams(5*time.Second, 5*time.Minute)
-		scaler.Scale(f.Namespace.Name, job.Name, uint(endParallelism), nil, waitForScale, waitForReplicas)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Ensuring active pods == endParallelism")
-		err = waitForAllPodsRunning(f.ClientSet, f.Namespace.Name, job.Name, endParallelism)
-		Expect(err).NotTo(HaveOccurred())
-	})
-
-	It("should scale a job down", func() {
-		startParallelism := int32(2)
-		endParallelism := int32(1)
-		By("Creating a job")
-		job := newTestJob("notTerminate", "scale-down", v1.RestartPolicyNever, startParallelism, completions)
-		job, err := createJob(f.ClientSet, f.Namespace.Name, job)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Ensuring active pods == startParallelism")
-		err = waitForAllPodsRunning(f.ClientSet, f.Namespace.Name, job.Name, startParallelism)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("scale job down")
-		scaler, err := kubectl.ScalerFor(batchinternal.Kind("Job"), f.InternalClientset)
-		Expect(err).NotTo(HaveOccurred())
-		waitForScale := kubectl.NewRetryParams(5*time.Second, 1*time.Minute)
-		waitForReplicas := kubectl.NewRetryParams(5*time.Second, 5*time.Minute)
-		err = scaler.Scale(f.Namespace.Name, job.Name, uint(endParallelism), nil, waitForScale, waitForReplicas)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Ensuring active pods == endParallelism")
-		err = waitForAllPodsRunning(f.ClientSet, f.Namespace.Name, job.Name, endParallelism)
-		Expect(err).NotTo(HaveOccurred())
-	})
-
-	It("should delete a job", func() {
```

Comment:
I'd like this one to be left until we have a server deletion in-place and coverage. IIRC this is not tested anywhere else.